src/Math/Angle.cs

```diff
@@ -351,6 +350,12 @@ public override int GetHashCode()
 
         #endregion
 
+        #region String
+
+        public override string ToString() => $"{radians} rad";
```

Comment:
I think these should return degrees for better human readability. Alternatively, consider extracting the factor of pi?